next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
```

Comment:
we need an image optimization solution. for now i've turned optimization off. any suggestions?